app/controllers/admin/invites_controller.rb

```diff
@@ -0,0 +1,64 @@
+module Admin
+  class InvitesController < Admin::BaseController
+    load_and_authorize_resource :conference, find_by: :short_title
+
+    def index
+      @invites = Invite.where(conference_id: @conference.id)
+      @invites.each do |invite|
+        invite.emails = User.find(invite.user_id).email
+      end
+    end
+
+    def new
+      @invite = @conference.invites.new
+    end
+
+    def create
+      emails_array = invite_params[:emails].split(',')
+      flag = 0
+      emails_array.each do |email|
+        @invite = @conference.invites.new(invite_params)
+        new_user = User.find_by(email: email)
+        new_user = User.invite!({ email: email }, current_user) if new_user.nil?
+        @invite.user_id = new_user.id
+        unless @invite.save
+          flag = 1
+          flash.now[:error] = 'Invitation failed due to some reasons.'
```

Comment:
Perhaps show the reasons? 
(same for update action)

Reply:
Alright!